src/SharedPipelineComponents/Datalock/DeployPeriodEnd/sql/ddl/PeriodEnd.Deds.DataLock.DDL.sprocs.sql

```diff
@@ -11,4 +11,31 @@ IF EXISTS(select 1 from sys.procedures where name = 'CleanupDeds')
 BEGIN
     DROP PROCEDURE DataLock.CleanupDeds
 END
+GO
```

Comment:
This seems like a change in functionality. Losing the current period filters?

Reply:
Oh, I see them. @MichaelYoung1981 can you address?

Reply:
Gary has confirmed I was wrong!  I've changed it to as per production and will create a story to remove linked server delete.